app/models/claims/allocation_filters.rb

```diff
@@ -37,12 +37,12 @@ def lgfs_scopes
     end
 
     def all_fixed_fee
-      where('"claims"."case_type_id" IN (?) OR "claims"."allocation_type" = ?', CaseType.fixed_fee.pluck(:id), 'Fixed')
+      where('"claims"."case_type_id" IN (?) OR "claims"."allocation_type" = ?', CaseType.fixed_fee.select(:id), 'Fixed')
```

Comment:
the PluckInWhere cop seems to be autocorrecting non-constant calls to pluck too which it describes as non-default

> This cop has two different enforcement modes. When the EnforcedStyle is conservative (the default) then only calls to pluck on a constant (i.e. a model class) in the where is used as offenses.

I am a little wary of this and not too sure of the imapct

Reply:
I haven't looked at the code too closely but I would be inclined to change this (and other examples you have highlighted) to something like this:

```ruby
  where(case_type: CaseType.fixed_fee).or(where(allocation_type: 'Fixed'))
```

This would remove the issue of `pluck` or `select` completely but I'd want to understand why it was done the way it is first.

Reply:
👍 yes, i like that.